hello-buck-cxx/hello-string.cxx

```diff
@@ -1,7 +1,13 @@
+#ifdef  BUCKSTRING
```

Comment:
nit: It's weird that this repo uses cxx for this filename. We tend to use cpp, others use cc :/